flows/definition/results_info.go

```diff
@@ -4,32 +4,16 @@ import (
 	"github.com/nyaruka/goflow/utils"
 )
 
+// holds information about a possible result in the flow
 type resultInfo struct {
-	Names []string `json:"names"`
+	Name string `json:"name"`
+	Key  string `json:"key"`
```

Comment:
@nicpottier doing no merging here across same labels etc in case in future we want to add node specific stuff like uuid or inferred type